modules/module-headings.php

```diff
@@ -91,6 +91,11 @@ function jetpack_get_module_i18n( $key ) {
 				'description' => _x( 'Write posts or pages in plain-text Markdown syntax.', 'Module Description', 'jetpack' ),
 			),
 
+			'masterbar' => array(
+				'name' => _x( 'WordPress.com Toolbar', 'Module Name', 'jetpack' ),
+				'description' => _x( 'An admin bar providing similar experience to the front-end navigation on WordPress.com.', 'Module Description', 'jetpack' ),
```

Comment:
Replaced it with the first suggestion in 0660ea26a58439a22995179e04b38be434d821f6.